src/main.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef MAIN
+#define MAIN
+
+#include <chrono>           /* Time */
+#include <iostream>         /* cout, ... */
+#include <fstream>          /* ifstream, ... */
+#include <math.h>           /* ceil, ... */
+#include <vector>           /* vector, ... */
+#include <sstream>
+#include <map>
+#include <algorithm>        /* std::remove */
+#include <iomanip>          /* std::setw() */
+#include <sys/stat.h>       /* chmod */
+
+
+typedef std::chrono::time_point<std::chrono::high_resolution_clock> time_point;
+
+
+/* Classes and structures */
+struct filePaths_ {
```

Comment:
Can we make these member variables static? Initialisation of non-static class or struct variables is a C++11 feature. If that's not possible then it's fine but we should add the C++11 requirement to the makefile

Reply:
Bear with me, I'm really just a pragmatic "programmer", so I'm not entirely sure what implications that would have. As far as I understand it, static variables will remember changes. So if I have two instances of _filePaths__ struct, and I change a static variable in the one instance, it will also be changed in the other instance?
I guess all variables in all structs would need to be static then?
That wouldn't be a problem until I want to use multiple instances of the molecule or fibril struct, which is actually the main reason I implemented structs, so I can use multiple instances.

On the other hand, you say that adding C++11 requirement to the makefile would be okay? Can we then just do that and maybe you can explain to me at some point if I understood static variables correctly ^^

Reply:
I added c++11 requirement to Makefile, should this be a problem, let me know.